src/components/shareModal/ShareModal.tsx

```diff
@@ -18,6 +18,8 @@ const ShareModal: FC<Props> = ({
   openModal,
   onClickCopyToClipboard,
   stickToBottomMobile = true,
+  showShareToFeed,
+  onclickShareToFeed,
```

Comment:
onclickShareToFeed => onClickShareToFeed (capital C in click)

Reply:
Done.